tm2/pkg/crypto/keys/client/common.go

```diff
@@ -25,11 +25,19 @@ var DefaultBaseOptions = BaseOptions{
 func HomeDir() string {
 	// if environment variable is set, always use that.
 	// otherwise, use config dir (varies depending on OS) + "gno"
-	var err error
-	dir := os.Getenv("GNO_HOME")
+
+	dir := os.Getenv("GNOHOME")
```

Comment:
TM2 shouldn't be aware of GNOHOME etc; let's pass the location from the caller, directly in gno.land/cmd/gnokey.